pkg/ccl/backupccl/backup_test.go

```diff
@@ -8452,17 +8452,18 @@ func TestIncorrectAccessOfFilesInBackupMetadata(t *testing.T) {
 	sqlDB.ExpectErr(t, "assertion: this placeholder legacy Descriptor entry should never be used", `RESTORE DATABASE r1 FROM LATEST IN 'nodelocal://0/test' WITH new_db_name = 'r2'`)
 }
 
-func TestManifestTooNew(t *testing.T) {
+// TestRestoringAcrossVersions test that users are only allowed to restore
```

Comment:
nit in the release note of the commit message: you're missing a couple words :D

Reply:
I think I fixed it but let me know if it still doesn't make sense.

Reply:
heh, i meant there was a small typo, but thank you for adding _many_ more words  :D